packages/client/src/index.ts

```diff
@@ -11,25 +11,27 @@ export type DecryptedMessage = {
 };
 
 export function storeSignature(
+  key: string,
   properties: Property[],
   sign: Signature,
   storage: StorageType,
   password?: string,
 ): void {
-  const key = getGeneratedRawKey(storage, password);
-  const encrypted = encryptMessage(JSON.stringify({ properties, sign }), key);
-  storeData(encrypted, DataType.Message, storage);
+  const encryptionKey = getGeneratedRawKey(key, storage, password);
```

Comment:
Change the name to distinguish the encryption key and the key among the key and value.